packages/starknet-snap/test/utils.ts

```diff
@@ -121,3 +125,135 @@ export async function generateAccounts(
   }
   return accounts;
 }
+
+/**
+ * Method to generate transactions.
+ *
+ * @param params
+ * @param params.chainId - Starknet Chain Id.
+ * @param params.address - Address of the account.
+ * @param params.contractAddresses - Contract addresses to generate transactions.
+ * @param params.txnTypes - Array of transaction types.
+ * @param params.finalityStatuses - Array of transaction finality status.
+ * @param params.executionStatuses - Array of transaction execution status.
+ * @param params.cnt - Number of transaction to generate.
+ * @returns An array of StarknetAccount object.
```

Comment:
Do we really return an array of StarknetAccount object ?? Or an array of transactions ?

Reply:
yes it should be transactions